### PR TITLE
Use secret in configuration for login cookie

### PR DESCRIPTION
**Pull request by kashizui**
This will allow cookies to work for multiprocess server.

@klopyrev @percyliang

### CODE REVIEW

File: codalab/lib/codalab_manager.py [hunk 1]
```diff
@@ -223,6 +223,11 @@ def init_config(self, dry_run=False):
             config['server']['engine_url'] = "sqlite:///{}".format(sqlite_db_path)
             print "Using SQLite database at: {}".format(sqlite_db_path)
 
+        # Generate secret key
+        config['server']['secret_key'] = get_random_string(
+            48, "=+/abcdefghijklmnopqrstuvwxyz"
+                "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
```
Comment (kashizui) [inline codalab/lib/codalab_manager.py:229]: How should we help people generate this if they already have a config file from a previous installation?

Comment (percyliang) [inline codalab/lib/codalab_manager.py:229]: This is a more general problem of updating the config file.  Perhaps there should be a separate cl command that reads in the config.json, adds fields that don't exist, and writes it back?

**Comment by percyliang** [comment]
LGTM

File: codalab/lib/codalab_manager.py (hunk 1 above)
Comment (kashizui) [inline codalab/lib/codalab_manager.py:229]: I was considering that too. If that makes sense, we can go ahead and add that.

Comment (klopyrev) [inline codalab/lib/codalab_manager.py:229]: -1 to feature requests not critical to migrating authentication as soon as
possible.

**Pull request merged**